packages/bot/src/stores/run-panel-store.js

```diff
@@ -11,9 +11,11 @@ export default class RunPanelStore {
         observer.register('bot.stop', this.onBotStopEvent);
         observer.register('contract.status', this.onContractStatusEvent);
         observer.register('bot.contract', this.onBotContractEvent);
+
+        this.setContractStage(CONTRACT_STAGES.not_running);
     }
 
-    @observable contract_stage = CONTRACT_STAGES.not_running‌;
+    @observable contract_stage = {};
```

Comment:
Why is `contract_stage` changed into an object?

Reply:
It's originally an object. Please look at `CONTRACT_STAGES`